test_open_file.py

```diff
@@ -0,0 +1,34 @@
+from eventio import Eventio
+from os import path
+from itertools import zip_longest
+
+TESTFILE = "test_files/corsika_iact/one_shower.dat"
+
+
+def test_can_open_file():
+    Eventio.from_file(TESTFILE)
+
+
+def test_file_has_objects_at_expected_position():
+    expected = [
+        (16, 1096),
+        (1128, 448),
+        (1592, 20),
+        (1628, 1096),
+        (2740, 16),
+        (2772, 6136),
+        (8924, 1096),
+        (10036, 16),
+    ]
+    f = Eventio.from_file(TESTFILE)
+
+    for o, (addr, size) in zip_longest(f.objects, expected):
+        # assert o.object.header.content_address == addr
```

Comment:
I don't know how to get the address

Reply:
Adress is the byte offset in the stream where the object starts, not sure if it's possible to get that with kaitai

Reply:
You can run the compiler with the `--read-pos` option, which will add a `_debug` field to the parsed objects that contains `start` and `end` markers:

```

In [9]: f.objects[0]._debug
Out[9]: 
defaultdict(dict,
            {'sync_marker': {'start': 0, 'end': 4},
             'object': {'start': 4, 'end': 1112}})
```

Reply:
```

In [21]: def get_payload_address(obj):
    ...:     if "subobjects" in obj._debug:
    ...:         return obj._debug["subobjects"]["start"]
    ...:     else:
    ...:         return obj._debug["payload"]["start"]
    ...: 

In [22]: [get_payload_address(o.object) for o in f.objects]
Out[22]: [16, 1128, 1592, 1628, 2740, 2772, 8924, 10036]
```